suite-native/transactions/src/components/TransactionsList/TransactionListItem.tsx

```diff
@@ -72,31 +86,27 @@ export const TransactionListItem = ({
     isFirst = false,
     isLast = false,
 }: TransactionListItemProps) => {
-    const includedCoinsCount = areTokensIncluded ? transaction.tokens.length : 0;
-
     const isTokenOnlyTransaction =
         areTokensIncluded && transaction.amount === '0' && transaction.tokens.length !== 0;
 
-    if (isTokenOnlyTransaction)
-        return (
-            <TokenTransferListItem
-                transaction={transaction}
-                accountKey={accountKey}
-                txid={transaction.txid}
-                tokenTransfer={transaction.tokens[0]}
-                includedCoinsCount={transaction.tokens.length - 1}
-            />
-        );
+    const symbolOrTokenTransfer = isTokenOnlyTransaction
```

Comment:
why is this needed? IMO if you pass both tokenTransfer and networkSymbol is should behave the same. Seems overcomplicated to me.

Reply:
`TransactionListItemContainerProps` `RequireExactlyOne` of those two and differentiates based on this

Reply:
but I will update this with TokenTransferListItem

Reply:
9e32cee8e9